sharepoint/sharepoint-ps/sharepoint-online/Get-SPOCrossTenantCompatibilityStatus.md

```diff
@@ -0,0 +1,63 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spocrosstenantcompatibilitystatus
+applicable: SharePoint Online
+title: Get-SPOCrossTenantCompatibilityStatus
+schema: 2.0.0
+author: trent-green
+ms.author: trgreen
+ms.reviewer:
+---
+
+# Get-SPOCrossTenantCompatibilityStatus
+
+## SYNOPSIS
+
+Determines the compatibility with the partner tenant.
+
+## SYNTAX
+
+```powershell
+Get-SPOCrossTenantCompatibilityStatus -PartnerCrossTenantHostUrl <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet allows you to determine the compatibility with the partner tenant before cross-tenant migration. You must be a SharePoint Online administrator or a global administrator to run it.
```

Comment:
>SharePoint Online administrator or a global administrator [](http://example.com/codeflow?start=123&length=57)

Here and other places: please use the official role names "SharePoint Administrator or a Global Administrator" as these are the formal Microsoft Entra built-in role names

[https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference](https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference)